pypeit/metadata.py

```diff
@@ -1388,6 +1400,7 @@ def write_sorted(self, ofile, overwrite=True, ignore=None, write_bkg_pairs=False
         ff.write('##end\n')
         ff.close()
 
+    # TODO: Do we need a calib file?
```

Comment:
It is only used by QA, but is used there quite a bit.

There might be a way to use just the `pypeit` file instead.
Staring at it, I think it can be done.  I suggest you raise an 
Issue requesting we do so.

Reply:
Added #1051 .